apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -163,6 +167,13 @@ export const getThemeConfig = (
   theme?: string,
   organization?: Organization | null,
 ): { themeConfig: Partial<LayoutProps> } => {
+  const organizationNamespace = JSON.parse(
+    organization?.namespace?.fields ?? '{}',
```

Comment:
Could we add an || instead of the ?? to handle the case if there is an empty string